### PR TITLE
Add network interface collection#119

### LINKED ISSUE #119: Implement NetworkInterfaceCollection
As oneview-redfish-user, I access "/redfish/v1/Systems/<server_hardware_uuid>/NetworkInterfaces" and receive the collection of NetworkInterface objects associated with this OneView's Server Hardware. There should be a NetworkInterface correspondent to each device at ServerHardware's 'portMap'. The device deviceNumber should be used as NetworkInterface ID.

### PR DESCRIPTION
**Pull request by ffdarkpenguin**
closes #119

### CODE REVIEW

File: oneview_redfish_toolkit/blueprints/network_interface_collection.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+import logging
+
+network_interface_collection = Blueprint(
+    "network_interface_collection", __name__)
+
+
+@network_interface_collection.route(
+    "/redfish/v1/Systems/<uuid>/NetworkInterfaces", methods=["GET"])
+def get_storage_collection(uuid):
```
Comment (danielreed) [inline oneview_redfish_toolkit/blueprints/network_interface_collection.py]: get_network_interface_collection

Comment (ffdarkpenguin) [reply oneview_redfish_toolkit/blueprints/network_interface_collection.py]: fixed

File: oneview_redfish_toolkit/blueprints/network_interface_collection.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+from oneview_redfish_toolkit import util
+
+import logging
+
+network_interface_collection = Blueprint(
+    "network_interface_collection", __name__)
+
+
+@network_interface_collection.route(
+    "/redfish/v1/Systems/<uuid>/NetworkInterfaces", methods=["GET"])
```
Comment (llvieira) [inline oneview_redfish_toolkit/blueprints/network_interface_collection.py]: "/redfish/v1/Systems/<uuid>/NetworkInterfaces" change to "/redfish/v1/Systems/<uuid>/NetworkInterfaces/"

File: oneview_redfish_toolkit/blueprints/network_interface_collection.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                'OneView Exception while looking for '
+                'server hardware: {}'.format(e)
+            )
+            abort(status.HTTP_500_INTERNAL_SERVER_ERROR)
+        else:
+            logging.error('Unexpected OneView Exception: {}'.format(e))
+            abort(status.HTTP_500_INTERNAL_SERVER_ERROR)
+    except Exception as e:
+        # In case of error print exception and abort
+        logging.error('Unexpected error: '.format(e))
```
Comment (llvieira) [inline oneview_redfish_toolkit/blueprints/network_interface_collection.py]: Missing the {}. The correct is: logging.error('Unexpected error: {}'.format(e))

**Pull request merged**